omegaconf/base.py

```diff
@@ -403,30 +398,65 @@ def _resolve_interpolation(
             try:
                 # NB: `match_list` is ignored here as complex interpolations may be
```

Comment:
what is NB ?

Reply:
Nota Bene (https://www.urbandictionary.com/define.php?term=NB)
Will replace with "Note"

Reply:
Done in 088f28a